test/Constraints/dynamic_lookup.swift

```diff
@@ -208,7 +208,7 @@ class Z4<T> where T : AnyObject { }
 // Don't allow one to call instance methods on the Type via
 // dynamic method lookup.
 type(of: obj).foo!(obj)(5)
-// expected-error@-1 {{instance member 'foo' cannot be used on type 'Id' (aka 'AnyObject')}}
+// expected-error@-1 {{instance member 'foo' of type 'Id' cannot be used in static context 'Id' (aka 'AnyObject')}}
```

Comment:
Looks like `'Id' (aka 'AnyObject')` is extraneous

Reply:
fixed